TestCases/parallel_regression.py

```diff
@@ -1632,6 +1632,18 @@ def main():
     ## Species solver ###
     #####################
 
+    # 2 species (1 eq) primitive venturi mixing
+    species2_primitiveVenturi           = TestCase('species2_primitiveVenturi')
+    species2_primitiveVenturi.cfg_dir   = "species_transport/venturi_primitive_3species"
+    species2_primitiveVenturi.cfg_file  = "species2_primitiveVenturi_mixingmodel.cfg"
```

Comment:
As pedro mentioned: you need a distinct name ... and the cfg needs to be added here